incubator/local-volume-provisioner/templates/namespace.yaml

```diff
@@ -0,0 +1,10 @@
+{{- if .Values.common.createNamespace }}
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: {{ .Values.common.namespace | quote }}
+  labels:
+    heritage: {{ .Release.Service | quote }}
```

Comment:
please use the new labels
see: https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md#metadata